app/models/taxonomies/location.rb

```diff
@@ -9,7 +9,7 @@ class Location < Taxonomy
   before_destroy EnsureNotUsedBy.new(:hosts)
 
   has_many :location_parameters, :class_name => 'LocationParameter', :foreign_key => :reference_id, :dependent => :destroy, :inverse_of => :location
-  has_many :default_users,       :class_name => 'User',              :foreign_key => :default_location_id
+  has_many :default_users,       :class_name => 'User',              :foreign_key => :default_location_id, :inverse_of => :location
```

Comment:
Should be `default_location`, will change when tests are finished. Please don't merge yet.